tests/db_functions/math/test_pi.py

```diff
@@ -0,0 +1,14 @@
+import math
+
+from django.db.models.functions import Pi
+from django.test import TestCase
+
+from ..models import FloatModel
+
+
+class PiTests(TestCase):
+
+    def test(self):
+        FloatModel.objects.create(f1=2.5, f2=15.9)
+        obj = FloatModel.objects.annotate(pi=Pi()).first()
+        self.assertAlmostEqual(obj.pi, math.pi, places=5)
```

Comment:
Is this not accurate to more than 5 decimal places on all backends? It is not that important, I am just surprised.

Reply:
I cannot remember whether I placed it on purpose or not. I removed it to give a try.